hack/test/e2e-qemu.sh

```diff
@@ -173,19 +164,12 @@ function create_cluster {
     --cidr=172.20.1.0/24 \
     --user-disk=/var/lib/extra:100MB \
     --user-disk=/var/lib/p1:100MB:/var/lib/p2:100MB \
-    --install-image=${INSTALLER_IMAGE} \
+    --install-image="${INSTALLER_IMAGE}" \
     --with-init-node=false \
-    --cni-bundle-url=${ARTIFACTS}/talosctl-cni-bundle-'${ARCH}'.tar.gz \
+    --cni-bundle-url="${ARTIFACTS}/talosctl-cni-bundle-\${ARCH}.tar.gz" \
```

Comment:
I'm not bash expert so is there any reason for `\` before `${ARCH}` here?

Reply:
so that it does not expand, the cli expect `${ARCH}` as is and not expanded

Reply:
Ah! Can't we use single quotes here tho?

Reply:
since the whole expression is inside a double quote, it would still expand